### PR TITLE
[FEAT] Jedi autocomplete

### PR DESCRIPTION
**Pull request by hv7214**
**Details**
1. Connected back Jedi autocompletion.
2. Added Jedi UNIX support.
3. Fixed crashing.

### CODE REVIEW

File: PythonAccess/jedi.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ QStringList Jedi::AutoComplete(const QString& code, long row, long col) {
     PyGILState_STATE d_gstate;
     d_gstate = PyGILState_Ensure();
     // Build the name object
-    const char* pythonCode = code.toStdString().c_str();
```
Comment (hv7214) [inline PythonAccess/jedi.cpp:22]: This was the bug. Somehow if you try invalid prefix then `pythonCode` will be pointing to some random address.

Comment (JaDogg) [reply PythonAccess/jedi.cpp:22]: Looks good can you add a comment for this.

Comment (JaDogg) [reply PythonAccess/jedi.cpp:22]: Also check if you type things like strings with Unicode letters (Hindi, Tamil, Chinese, etc..) that this works as well.

File: ep_jedi.py [hunk 2]
```diff
@@ -14,7 +14,16 @@
 def get_completions(source, row, col, script_path=""):
     return []
 
-
+def real(source, row, col, script_path=""):
+    completions = []
```
Comment (JaDogg) [inline ep_jedi.py:18]: You might need to import `jedi` again in this function? Does this work in Linux 🤔

Comment (hv7214) [reply ep_jedi.py:18]: yes, it do work

File: PythonAccess/jedi.cpp (hunk 1 above)
Comment (hv7214) [reply PythonAccess/jedi.cpp:22]: Sure

File: ep_jedi.py (hunk 2 above)
Comment (hv7214) [reply ep_jedi.py:18]: infact my env is Ubuntu 20.04 ; )

File: PythonAccess/jedi.cpp (hunk 1 above)
Comment (hv7214) [reply PythonAccess/jedi.cpp:22]: > Also check if you type things like strings with Unicode letters (Hindi, Tamil, Chinese, etc..) that this works as well.

You mean like this `someOtherLanguageText` then space + ctrl

Comment (hv7214) [reply PythonAccess/jedi.cpp:22]: Oh, I see, it's not working. Understood it is encoding issue

**Pull request merged**